rampwf/score_types/detection.py

```diff
@@ -652,6 +647,79 @@ def precision_recall_curve(y_true, y_pred, conf_thresholds, iou_threshold=0.5):
     return np.array(ps), np.array(rs)
 
 
+def _add_id(y):
+    """Helper function to flatten and add id column to list of lists"""
+    y_new = []
+
+    for i, y_patch in enumerate(y):
+        if len(y_patch):
+            tmp = np.asarray(y_patch)
+            tmp = np.insert(tmp, 0, i, axis=1)
+            y_new.append(tmp)
+
+    return np.vstack(y_new)
+
+
+def precision_recall_curve_greedy(y_true, y_pred, iou_threshold=0.5):
+    """
+    Calculate precision and recall incrementally based on the predictions
+    sorted by confidence (not calculating an exact optimal match for each
+    confidence level).
+
+    Parameters
+    ----------
+    y_true
+    y_pred
+    iou_threshold
```

Comment:
Mention here the expected input for y_pred, since its different than y_true 
=> `(conf, row, col, rad)`